openstack/image/api.go

```diff
@@ -253,6 +267,8 @@ func (h APIHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 // on return values all the time.
 func errorResponse(err error) APIResponse {
 	switch err {
+	case ErrNoImage:
+		return APIResponse{http.StatusNotFound, nil}
 	default:
```

Comment:
@kaccardi, this error is going to be used when you're using the Get method. If the image is not Found, it's getting a 404 error, which is the http.StatusNotFound 
https://golang.org/pkg/net/http/
```
  (StatusNotFound                     = 404 // RFC 7231, 6.5.4)
```

The idea of this errorResponse function (AFAIK) is to be the one for handling the different HTTP errors. For now those are the implemented ones, expect more to be added.